### PR TITLE
admin/serve-doc: Switch to python3 only

### PR DESCRIPTION
**Pull request by badone**
Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: admin/serve-doc [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 from __future__ import print_function
 
-import SimpleHTTPServer
-import SocketServer
+try:
+    # Python 3
+    import http.server as server
+    import socketserver
+except ImportError:
+    # Python2
```
Comment (tchaikov) [inline admin/serve-doc]: can we just drop python2 support?

Comment (badone) [reply admin/serve-doc]: @tchaikov I wasn't brave enough to make that call. Seems every time I do it comes back to bite me so I went out of my way to make it backwards compatible. Are we definitely py3 only and completely on master now?

Comment (badone) [reply admin/serve-doc]: will do

**Comment by tchaikov** [comment]
```
[ RUN      ] TestDeepCopy.Snaps_SmallerDstObjSize
creating image with stripe unit: 2097152, stripe count: 16
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/librbd/test_DeepCopy.cc:107: Failure
Expected equality of these values:
  read_size
    Which is: 4194304
  m_src_ictx->io_work_queue->read( offset, read_size, librbd::io::ReadResult{src_result}, 0)
    Which is: 2097152
[  FAILED  ] TestDeepCopy.Snaps_SmallerDstObjSize (414 ms)
```

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by badone** [comment]
Weird, trying a rebase.

**Comment by badone** [comment]
Rebase seems to have worked.

**Pull request merged**